install

```diff
@@ -12,7 +13,7 @@ function install_env {
     tk apply environments/$ENV                                      # Installs everything for your environment.
 }
 
-if [[ $(docker info --format "{{.MemTotal}}") -lt 2560000000 ]]; then
+if [[ "$MODE" != "cloud" ]] && [[ $(docker info --format "{{.MemTotal}}") -lt 2560000000 ]]; then
```

Comment:
Could you add a note to the README (wherever it makes sense) that this is an option?

Reply:
I realize this is for Grafana Cloud but there doesn't seem to be anything cloud-specific about this. Maybe a different name like "app-only" or "existing" or something like that? (please feel free to pick something better than my suggestions)